doc/examples/filters/plot_deconvolution.py

```diff
@@ -19,23 +19,32 @@
 
 import numpy as np
 import matplotlib.pyplot as plt
+import skimage as ski
 
 from scipy.signal import convolve2d as conv2
 
-from skimage import color, data, restoration
 
 rng = np.random.default_rng()
 
-astro = color.rgb2gray(data.astronaut())
+# Convert astronaut image to grayscale
+astro = ski.color.rgb2gray(ski.data.astronaut())
 
+# Define PSF
 psf = np.ones((5, 5)) / 25
-astro = conv2(astro, psf, 'same')
-# Add Noise to Image
-astro_noisy = astro.copy()
-astro_noisy += (rng.poisson(lam=25, size=astro.shape) - 10) / 255.0
 
-# Restore Image using Richardson-Lucy algorithm
-deconvolved_RL = restoration.richardson_lucy(astro_noisy, psf, num_iter=30)
+# Convolve image with the PSF to simulate a blurred image
+astro_blurred = conv2(astro, psf, 'same')
+
+# Introduce poisson noise to the blurred image (https://en.wikipedia.org/wiki/Image_noise)
```

Comment:
```suggestion
# Add Poisson noise to the blurred image
```
I would point to the main article (https://en.wikipedia.org/wiki/Shot_noise) or, rather, let the interested reader go to: https://numpy.org/doc/2.0/reference/random/generated/numpy.random.Generator.poisson.html

Reply:
done!